stakercfg/config.go

```diff
@@ -35,6 +37,14 @@ const (
 	// we risk into having transactions rejected by the network due to low fee.
 	DefaultMinFeeRate = 2
 	DefaultMaxFeeRate = 200
+
+	// JSON-RPC server config
+	defaultMaxOpenConnections  = 0 // unlimited
+	defaultReadTimeout         = 15 * time.Second
```

Comment:
I would set `defaultReadTimeout` and `defaultWriteTimeout` 2min. In some operations we are waiting for thransaction inclusion in Babylon block so it can take a bit of time